### DISCUSSION TITLE
Hibernation - Why is it disabled? How to enable it back?

### DISCUSSION THREAD

**Discussion by MicaelJarniac** [memstechtips/UnattendedWinstall#77] [Q&A] [answered]
I noticed that the scripts here disable hibernation.

I wanted to know why. What makes hibernation bad?

I also wanted to know how to enable it back, after Windows is already installed. Is there any specific process, or can I just search online?

Also, on a laptop, would it be better to have hibernation enabled? If it can't hibernate, won't it use more battery with the lid closed?

**Comment by marcelkb** [comment]
had the same question in mind.
my guess is that the hibernate function needs the same amount of storage place as your ram is big. So if you have f.e. 32gb ram, 32gb is needed on your storage and reserved for it.

the script does
reg.exe add "HKLM\System\CurrentControlSet\Control\Session Manager\Power" /v HibernateEnabled /t REG_DWORD /d 0 /f
reg.exe add "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\FlyoutMenuSettings" /v ShowHibernateOption /t REG_DWORD /d 0 /f

so replacing /d 0  with /d 1 and executing it should enable it back

**Comment by memstechtips** [comment, marked as answer]
Yes, you should be able to enable it back as described by @marcelkb 

Regarding why it's disabled in the first place, it is one of the tweaks in the Chris Titus Windows Utility.
I didn't really give it much thought when I included it in the file to be honest.
